string object_name
float32 h_value